sympy/stats/frv.py

```diff
@@ -146,10 +146,10 @@ def __new__(cls, domain, condition):
         # Check that we aren't passed a condition like die1 == z
         # where 'z' is a symbol that we don't know about
         # We will never be able to test this equality through iteration
```

Comment:
@czgdp1807 this seems to be the explanation for the exception.

Reply:
Okay, I will add a test for it so that if someone in future tries to remove this part then at least a test failure will avoid it.